src/isString.ts

```diff
@@ -0,0 +1,3 @@
+export const isString = (_: unknown): boolean => {
+	return Object.prototype.toString.call(_) === '[object String]';
```

Comment:
Why not use `typeof _ === 'string'`. Is it just to catch `typeof new String('abc') === 'object'` ?

Reply:
yeah exactly

Reply:
got it from https://stackoverflow.com/a/17772086 :)